zlux-file-explorer.ppf

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Similar to above, this file is a remnant of the **master** branch of FE, the widget version doesn't have this file. It could potentially work, but I'd recommend just taking it out for now, or asking Yan Li what these .ppf files are about (his commit is right here https://github.com/zowe/zlux-file-explorer/commit/bf99fb72227286991a58d905b7b61b248fde76a0)